source/ID/concordium-did.rst

```diff
@@ -0,0 +1,479 @@
+.. _concordium_did:
+
+==================================
+Concordium DID Method Version 1.0
+==================================
+
+About
+=====
+
+The Concordium DID method specification conforms to the requirements specified in `Decentralized Identifiers (DIDs) v1.0 <w3c-did-core-v1.0_>`_ [DID] published by the W3C Credentials Community Group.
+For more information about DIDs and DID method specifications, please see the `DID Primer`_ and `DID Spec`_.
+
+Abstract
+=========
+
+Concordium is a layer-1 blockchain with identity built into the core protocol.
+We distinguish the following types of DIDs:
+
+- **Account DID** refer to accounts on the Concordium blockchain.
+- **Smart Contract DID** refer to smart contracts instances on the Concordium blockchain.
+- **Public Key DID** refers to a subject that knows the corresponding secret key.
+
+Status of This Document
+=======================
+
+This is a draft document and may be updated.
+
+[Add more legal text here?]
+
+Specification
+=============
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in :rfc:`2119`.
+
+Concordium DID Identifier
+=========================
+
+Method Name
+-----------
+
+The Concordium DID method name is ``ccd``.
+A DID identifier that uses this method must begin with the following prefix: ``did:ccd``.
+Per DID specification [DID] this string must be in lowercase.
+
+Identifier Syntax
+-----------------
+
+Concordium DID identifiers are defined by the following ABNF_:
+
+.. code-block:: ABNF
+
+  ccd-did = prefix ":" *1(network ":") (acctype / pkctype / scitype)
+  prefix  = %s"did:ccd"
+  network = "testnet" / "mainnet"
```

Comment:
Not clear what exactly network identifier is and it would be nice to define it somewhere (for example, for "pkctype: did-s, I have no idea what network identifier would mean.)

Reply:
Yes, good point, I think we should make a separate case for `pkctype`, I agree that it's not clear what network means in this context.

Reply:
As we agreed, since the network is optional, we keep the syntax as it is, but add a note that if the network is present, it could affect the generated DID document in the future (e.g. different verification method).